lexicons/app/bsky/feed/generator.json

```diff
@@ -0,0 +1,29 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.generator",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "A declaration of the existence of a feed generator",
+      "key": "named",
```

Comment:
wasn't sure what to call this key type

other options I came up with: "custom", "provided", "user-provided", "unconstrained"

Note: I also opted to _not_ have a feed name in the record properties since it can be provided by the rkey

Reply:
not a huge question but some other options:

- string
- user-defined